src/formatter/json_formatter_spec.js

```diff
@@ -372,4 +373,235 @@ describe('JsonFormatter', () => {
       ])
     })
   })
+
+  describe('one scenario with one step with localization', () => {
```

Comment:
Can we please add the name of the language here is case this needs to be updated in the future?
```js
describe('one scenario with one step with localization (<language>)', () => {
```